jgrapht-core/src/main/java/org/jgrapht/alg/intervalgraph/IOrdering.java

```diff
@@ -0,0 +1,32 @@
+package org.jgrapht.alg.intervalgraph;
+
+import org.jgrapht.Graph;
+
+public class IOrdering<V>{
+
+    /**
+     * Calculates if the given sweep is an I-Ordering
+     * (according to the Graph graph)
+     *
+     * @param sweep the order we want to check if its an I-Order
+     * @param graph the graph we want to check if its an I-Order
+     * @return true, if sweep is an I-Order according to graph
+     */
```

Comment:
I think, this algorithm has a complexity of O(|sweep|^3) if containsEdge is in O(1). According to the paper, there should also be an algorithm for O(|V|+|E|). We should maybe consider thinking about how to implement a linear time i-order check? (at least after tuesday)